x-pack/beatless/config/config.go

```diff
@@ -2,11 +2,73 @@
 // or more contributor license agreements. Licensed under the Elastic License;
 // you may not use this file except in compliance with the Elastic License.
 
+// Config is put into a different package to prevent cyclic imports in case
+// it is needed in several locations
+
 package config
 
+import (
+	"fmt"
+
+	"github.com/elastic/beats/libbeat/common"
+)
+
+// ConfigOverrides overrides the defaults provided by libbeat.
+var ConfigOverrides = common.MustNewConfigFrom(map[string]interface{}{
+	"path.data":              "/tmp",
+	"path.logs":              "/tmp/logs",
+	"logging.to_stderr":      true,
+	"logging.to_files":       false,
+	"logging.level":          "debug",
+	"setup.template.enabled": true,
+	"queue.mem": map[string]interface{}{
+		"events":           50,
+		"flush.min_events": 1,
+		"flush.timeout":    "0.1s",
+	},
+})
```

Comment:
Yup, super hacky, but I haven't found a good way without that :(
I would have preferred to be able to define internal config type instead of providing an untyped common.Config. I expected the reference syntax to work.